include/Util/NamedPipeListener.h

```diff
@@ -58,31 +58,31 @@ NamedPipeListener<T>::~NamedPipeListener() {
 
 template <typename T>
 bool NamedPipeListener<T>::StartListening(const std::function<void(T*)>& callback) {
-  if (m_threadActive.exchange(true))
+  if (threadActive_.exchange(true))
     // Thread already running
     return false;
 
-  m_thread = std::thread(&NamedPipeListener<T>::ListenerThread, this, callback);
+  thread_ = std::thread(&NamedPipeListener<T>::ListenerThread, this, callback);
 
   return true;
 }
 
 template <typename T>
 void NamedPipeListener<T>::StopListening() {
-  if (m_threadActive.exchange(false))
+  if (threadActive_.exchange(false))
     // Thread running
-    m_thread.join();
+    thread_.join();
 }
 
 template <typename T>
 bool NamedPipeListener<T>::Connect(NamedPipeListenerData<T>* data) {
-  BOOL fConnected = ConnectNamedPipe(data->hPipeInst, &data->oOverlap);
-  if (!fConnected) {
+  if (const BOOL fConnected = ConnectNamedPipe(data->hPipeInst, &data->oOverlap); !fConnected) {
```

Comment:
I think that this can just be `if(ConnectedNamedPipe(...))`

Reply:
Yep